Sources/Sentry/include/SentryBinaryImageCache.h

```diff
@@ -0,0 +1,23 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BinaryImageInfo : NSObject
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic) uint64_t address;
+@property (nonatomic) uint64_t size;
+@end
+
+@interface SentryBinaryImageCache : NSObject
+
+@property (nonatomic, readonly, class) SentryBinaryImageCache *shared;
```

Comment:
`m`: As there can only be one instance of this, I think we should either make everything static here or add `SENTRY_NO_INIT` here.